src/main/java/org/mockito/ArgumentCaptor.java

```diff
@@ -134,6 +134,15 @@ public List<T> getAllValues() {
         return this.capturingMatcher.getAllValues();
     }
 
+    /**
+     * Get the raw class being captured.
+     *
+     * @return the raw class that is being captured by this captor.
+     */
+    public Class<? extends T> getCaptorType() {
```

Comment:
Do we need to expose this as public API or do we solely use this for testing? I think it is the latter (which is good), but then let's make this package-protected.

Reply:
Sounds good, updated the PR as suggested!